unit_testing/test_utils.py

```diff
@@ -7,6 +7,20 @@
 from spinalcordtoolbox import utils
 
 
+def test_add_suffix():
+    assert utils.add_suffix('t2.nii', '_mean') == 't2_mean.nii'
+    assert utils.add_suffix('t2.nii.gz', 'a') == 't2a.nii.gz'
```

Comment:
```suggestion
    assert utils.add_suffix('t2.nii.gz', 'a') == 't2a.nii.gz'
    assert utils.add_suffiix('var/lib/usr/t2.nii.gz', 'sfx') == 'var/lib/usr/t2sfx'.nii.gz'
    assert utils.add_suffiix('var/lib.version.3/usr/t2.nii.gz', 'sfx') == 'var/lib.version.3/usr/t2sfx'.nii.gz'
```

Reply:
done in 58e1d86e8e4c7cb1961a6e68b643947f7390854a